docs/resources/idp_saml_v2.md

```diff
@@ -45,6 +45,7 @@ resource "fusionauth_idp_saml_v2" "Saml" {
 * `debug` - (Optional) Determines if debug is enabled for this provider. When enabled, each time this provider is invoked to reconcile a login an Event Log will be created.
 * `domains` - (Optional) This is an optional list of domains that this OpenID Connect provider should be used for. This converts the FusionAuth login form to a domain-based login form. This type of form first asks the user for their email. FusionAuth then uses their email to determine if an OpenID Connect identity provider should be used. If an OpenID Connect provider should be used, the browser is redirected to the authorization endpoint of that identity provider. Otherwise, the password field is revealed on the form so that the user can login using FusionAuth.
 * `email_claim` - (Optional) The name of the email claim (Attribute in the Assertion element) in the SAML response that FusionAuth uses to uniquely identity the user. If this is not set, the `use_name_for_email` flag must be true.
+* `username_claim` - (Optional) The name of the username claim (Attribute in the Assertion element) in the SAML response that FusionAuth uses to uniquely identity the user.
```

Comment:
Suggestion to mirror the [API documentation](https://fusionauth.io/docs/apis/identity-providers/samlv2):
```suggestion
* `username_claim` - (Optional) The name of the claim in the SAML response that FusionAuth uses to identify the username. If this is not set, the NameId value will be used to link a user. This property is required when linkingStrategy is set to LinkByUsername or LinkByUsernameForExistingUser
```